hollow/src/main/java/com/netflix/hollow/core/HollowStateEngine.java

```diff
@@ -63,6 +63,11 @@ public interface HollowStateEngine extends HollowDataset {
      * */
     String HEADER_TAG_METRIC_ANNOUNCEMENT = "hollow.metric.announcement";
 
+    /**
+     * A header tag indicating which version is this snapshot produce to.
+     */
+    String HEADER_TAG_PRODUCER_TO_VERSION = "hollow.producer.to.version";
```

Comment:
nit: name `hollow.blob.to.version` so it is more clear that this is the one in the blob (differentiating the one in metadata)